http/preview.go

```diff
@@ -59,6 +59,20 @@ func handleImagePreview(w http.ResponseWriter, r *http.Request, file *files.File
 		return errToStatus(err), err
 	}
 
+	fd, err := file.Fs.Open(file.Path)
+	if err != nil {
+		return errToStatus(err), err
+	}
+	defer fd.Close()
+
+	if format == imaging.GIF && size == sizeBig {
+		_, err := rawFileHandler(w, r, file)
+		if err != nil {
+			return errToStatus(err), err
+		}
```

Comment:
```suggestion
		if _, err := rawFileHandler(w, r, file); err != nil { //nolint:govet
			return errToStatus(err), err
		}
```

Reply:
I've now set up the linter on my local environment. I'm curious why it complains about shadowing when the same thing seems to happen in L62 where Fs.Open overrides err. It doesn't complain there.

Also I'm still getting a warning of `WARN [runner/nolint] Found unknown linters in //nolint directives: mnd, shadow`. Any ideas? Or is this known behaviour?

Reply:
Just skip warnings for now